### ISSUE TITLE
Fix horizontal and tertiary nav to have keyboard navigation

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#4784]
**Describe the issue. What is the expected and unexpected behavior?**
The horizontal and tertiary navigation should be able to have keyboard focus. This also flags an axe issue, "Ensure that scrollable region has keyboard access":
<img width="1312" alt="Screen Shot 2020-09-04 at 1 47 07 PM" src="https://user-images.githubusercontent.com/28281340/92271747-596d1100-eeb6-11ea-8037-4e36da8e608b.png">

This is likely due to the fact that the anchor links don't have href attributes.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
https://www.patternfly.org/v4/documentation/react/components/nav#horizontal-only-in-pageheader
Try to navigate by keyboard Tab or open axe browser extension and run. 

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
